### ISSUE TITLE
Editor javadoc does not display parameter names

### ISSUE THREAD

**Issue by meatfighter** [apache/netbeans#6301]
### Apache NetBeans version

Apache NetBeans 18

### What happened

When I hold down Ctrl and hover the mouse cursor over a constructor or method, the javadoc that popups does not display the names of the variables properly. 

### How to reproduce

In a Java file, define a new BufferedImage(), hold down Ctrl, and hover the mouse cursor over the constructor name. It shows three parameters called i, i1, and i2, instead of width, height, and imageType.

### Did this work correctly in an earlier version?

Apache NetBeans 16

### Operating System

Windows 10 version 10.0 running on AMD64

### JDK

1.8.0_241

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by neilcsmith-net** [comment]
JDK 1.8 is no longer supported.  Please try running the IDE on a supported JDK and see if this can be reproduced.

**Comment by neilcsmith-net** [comment]
Do you have sources set in that platform configuration?

**Comment by PetrMiksik** [comment]
If you mean a source profile, yes. It's set to Full JRE. And platform sources are also set in the configuration ("C:\Program Files\Java\jdk1.8.0_231\src.zip" actually). Note that this does not occur for every classes in JDK 8. It seems that mainly "java.awt" package classes are affected.

**Comment by neilcsmith-net** [comment]
> And platform sources are also set in the configuration

Yes, that's what I meant.  I cannot reproduce this here, on Linux at least (I'll check Windows next time I'm on it).  What about build system, project type?  Which JDK distribution?

**Comment by PetrMiksik** [comment]
Hello,
the project type is Java with Ant (Java Application). Java platform and source in project configuration is set to 1.8, Full JRE profile.
Windows 10/11 -  NetBeans runs on OpenJDK 17 from Eclipse Temurin distribution, JDK8 is from Oracle.
Linux (Ubuntu 22.04) - canonical OpenJDK 17 is installed and JDK8 is also from Oracle.
No exception or error is thrown, even during startup. See the picture below.

![BufferedImage_JavaDoc](https://github.com/apache/netbeans/assets/106738725/fa2a13c9-8cd1-4676-8799-fbdd7e109652)

Also attaching IDE log here
[IDE.log](https://github.com/apache/netbeans/files/12288273/IDE.log)

**Comment by neilcsmith-net** [comment]
I cannot reproduce this (at least with OpenJDK 8) - maybe someone can test with Oracle.

Please try with a clean userdir and cache - eg. `netbeans --userdir /tmp/nb18` and without additional plugins or settings brought over from a previous installation.

**Comment by PetrMiksik** [comment]
That's fun . You are not able to reproduce it, but I cannot get rid of it :-)
Windows 11, clean installation of NetBeans 18 (I've deleted both userdir and cache dirs from previous installation), no 3rd party plugins and Open JDK 1.8 and 17 from Eclipse Adoptium installed - no Oracle distribution. I've created a new Java project (Java with Ant, Java Application), set Java platform to 1.8 and - no luck. The issue is still the same.

Attaching a new IDE log. Maybe you will find something suspicious.
[IDE log.txt](https://github.com/apache/netbeans/files/12289666/IDE.log.txt)

**Comment by PetrMiksik** [comment]
The problem is only with the popup javadoc window. If I open the class in the IDE editor and show the Javadoc window, everything is OK.

Popup window
![Javadoc - popup](https://github.com/apache/netbeans/assets/106738725/6b860a9e-39cb-44e3-b545-5af9e647cddd)

Generated javadoc from the opened source in the editor
![Javadoc - window](https://github.com/apache/netbeans/assets/106738725/2e3a9eb9-0b43-40dc-8e2d-255997018e14)

**Comment by t-gergely** [comment]
Classes `org.netbeans.api.java.source.ui.ElementJavadoc ` and `HTMLJavadocParser` seem to ignore the parameter names from the javadoc, so if your class file doesn't have the debug info (namely the LocalVariableTable AFAIK), you don't get the proper parameter names. It is a problem with `native` methods like `System.arraycopy (Object src, int srcPos, Object dest, int destPos, int length)` that shows as `arraycopy(Object o, int i, Object o1, int i1, int i2)`.
